### PR TITLE
Add more constants related to Somfy Thermostat

### PR DESCRIPTION
**Pull request by tetienne**
See https://github.com/home-assistant/core/pull/67169/files

### CODE REVIEW

File: pyoverkiz/enums/state.py [hunk 1]
```diff
@@ -124,12 +127,15 @@ class OverkizState(str, Enum):
...
     CORE_WIND_SPEED = "core:WindSpeedState"
+    FREEZE_MODE = "freezeMode"
+    FURTHER_NOTICE = "further_notice"
     HLRRWIFI_FAN_SPEED = "hlrrwifi:FanSpeedState"
     HLRRWIFI_LEAVE_HOME = "hlrrwifi:LeaveHomeState"
     HLRRWIFI_MAIN_OPERATION = "hlrrwifi:MainOperationState"
     HLRRWIFI_MODE_CHANGE = "hlrrwifi:ModeChangeState"
     HLRRWIFI_ROOM_TEMPERATURE = "hlrrwifi:RoomTemperatureState"
     HLRRWIFI_SWING = "hlrrwifi:SwingState"
+    INACTIVE = "inactive"
```
Comment (iMicknl) [inline pyoverkiz/enums/state.py:138]: Is this a state or a state param?

File: pyoverkiz/enums/state.py [hunk 2]
```diff
@@ -196,6 +203,7 @@ class OverkizState(str, Enum):
     RTDS_CONTROLLER_ORIGINATOR = "rtds:ControllerOriginatorState"
     RTDS_CONTROLLER_SENSING = "rtds:ControllerSensingState"
     RTDS_CONTROLLER_SIREN = "rtds:ControllerSirenState"
+    SLEEPING_MODE = "sleepingMode"
```
Comment (iMicknl) [inline pyoverkiz/enums/state.py:206]: This is also not in the right place?

Comment (tetienne) [reply pyoverkiz/enums/state.py:206]: Sorry, you are right. I misread your comment: https://github.com/iMicknl/python-overkiz-api/pull/492/files

Comment (iMicknl) [reply pyoverkiz/enums/state.py:206]: You can use command.py OverkizCommandParam.

File: pyoverkiz/enums/state.py (hunk 1 above)
Comment (tetienne) [reply pyoverkiz/enums/state.py:138]: This is a state value, sorry. We don’t use enum for state value, right?

Comment (tetienne) [reply pyoverkiz/enums/state.py]: I just checked, inactive and active are never used as command parameter, they are just a state value.

File: pyoverkiz/enums/state.py (hunk 2 above)
Comment (tetienne) [reply pyoverkiz/enums/state.py]: Done

File: pyoverkiz/enums/command.py [hunk 3]
```diff
@@ -155,8 +157,10 @@ class OverkizCommandParam(str, Enum):
     EXTERNAL_GATEWAY = "externalGateway"
     FAN = "fan"
     FREE = "free"
+    FREEZE_MODE = "freezeMode"
     FROSTPROTECTION = "frostprotection"
     FULL = "full"
+    FURTHER_NOTICE = "further_notice"
```
Comment (iMicknl) [inline pyoverkiz/enums/command.py:163]: Is this a Somfy inconsistency? Instead of `furtherNotice`.

**Review by iMicknl — APPROVED** [review]

File: pyoverkiz/enums/state.py (hunk 1 above)
Comment (iMicknl) [reply pyoverkiz/enums/state.py]: In theory we could still add it, so you can use the enum, however than the name OverkizCommandParam is a bit weird.

**Pull request merged**